definitions/browser-micro_frontend/definition.yml

```diff
@@ -0,0 +1,21 @@
+domain: BROWSER
+type: MICRO_FRONTEND
+goldenTags:
+  - type
+  - environment
+  - repository
+  - team
+synthesis:
+  rules:
+    - identifier: mfe.id
+      name: mfe.name
+      conditions:
+        - attribute: mfe.id
+          present: true
+        - attribute: mfe.name
+          present: true
```

Comment:
these are implicit already from having identifier and name.

Is there any other attribute we should use the make sure is applying to the correct events? maybe an eventType? a source?

Reply:
You're right. Thank you for pointing this out. I've updated the PR to use a unique `eventType` attribute in addition to the already implicit `mfe.id` and `mfe.name` attributes.